### PR TITLE
Add new columns and handle horizontal scroll in payment request table

### PR DESCRIPTION
**Pull request by sauravmaiti22**
- Added Order id and payment request id columns to the table which are not selected by default. Will appear once selected in the edit columns popover. 
- If the table overflows, the scrollbar would appear. It's a customised scrollbar.
- Made the last column with context menu sticky so it is always visible when the table is scrolled. 
- Added a gradient background to it.
- Adjusted sizes of the column so by default i.e without the extra columns, the scrollbar doesn't appear and it looks good atleast on a tablet.
- The changes are quite complicated. Needed a lot of adjustments to get this to work properly. Let me know if you see any issues.
- Moved the pager to the bottom as per new design. Have to make changes to the pager icon according to new design but that will be done in a separate task.
https://www.figma.com/file/Sc96Upmn3MpOmvM9jgvaUG/MMB-Portal-%2F-HiFi?type=design&node-id=11673-37090&mode=dev

### CODE REVIEW

**Comment by sipsorcery** [comment]
Latest changes for this PR can be tested at https://business-dev.nofrixion.com/181.

**Review by pabloi-nf — CHANGES_REQUESTED** [review]
Thanks Saurav. It works and looks beautifully.
The only thing I would adjust to make it perfect is to add a quick 100ms transition on the hover state of the scrollbar, so it doesn't jump so much.

Thanks

**Comment by sipsorcery** [comment]
Latest changes for this PR can be tested at https://business-dev.nofrixion.com/181.

**Comment by sauravmaiti22** [comment]
> Thanks Saurav. It works and looks beautifully. The only thing I would adjust to make it perfect is to add a quick 100ms transition on the hover state of the scrollbar, so it doesn't jump so much.
> 
> Thanks

@pabloi-nf Good point. I did notice it. I have added the transition now.

**Comment by agranillonf** [comment]
This was so fire I needed sunglasses to see it. We should sell it.

**Review by agranillonf — APPROVED** [review]

**Review by pabloi-nf — APPROVED** [review]

File: packages/components/src/components/ui/TableScrollbar/TableScrollbar.tsx [hunk 1]
```diff
@@ -32,7 +32,7 @@ const TableScrollbar = forwardRef(
                   height: '16px',
                 }}
                 className={cn(
-                  'hover:border-[#EAF0F1] hover:bg-[#73808C] border-[6px] border-[#F6F8F9] bg-[#ABB2BA] rounded-lg my-auto',
+                  'hover:border-[#EAF0F1] hover:bg-[#73808C] border-[6px] border-[#F6F8F9] bg-[#ABB2BA] rounded-lg my-auto transition-colors duration-[100ms] ease-out',
```
Comment (Publicker) [inline packages/components/src/components/ui/TableScrollbar/TableScrollbar.tsx:35]: Just a minor, nitpicky note – but that duration is already built into TailwindCSS classes. Whenever using arbitrary values, please check first to see if the class already exists.

**Pull request merged**